### PR TITLE
Migrate Profile index.js to TypeScript

### PR DESCRIPTION
**Pull request by Prakhar-Agarwal-byte**
**Fixes**

Fixes Part of #93 by @shubhanshu02

**Description**

**Checklist:**

-   [ ] I have read the contribution guidelines in the repository.
-   [ ] My branch is up-to-date with upstream branch.
-   [ ] My pull request has a descriptive title (not a vague title like "Update package.json").
-   [ ] My code follows the established code style of the repository.
-   [ ] I have tried running the project locally and verified that there are no visible errors.

### CODE REVIEW

File: pages/profile/[username].js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { FaGithub } from "react-icons/fa"
 
 import axios from "../../util/axios"
 import { useAuth } from "../../context/auth"
-import { Interests } from "../../components/Profile"
+import { Interests } from "../../components/Profile/index.tsx"
```
Comment (shubhanshu02) [inline pages/profile/[username].js:10]: This line change is not required.

**Review by shubhanshu02 — CHANGES_REQUESTED** [review]
Just a fix, rest everything looks good. And rebase.

File: pages/profile/[username].js (hunk 1 above)
Comment (Prakhar-Agarwal-byte) [reply pages/profile/[username].js]: Done

Comment (shubhanshu02) [reply pages/profile/[username].js]: Even `import { Interests } from "../../components/Profile"` should work fine :thinking:

Comment (Prakhar-Agarwal-byte) [reply pages/profile/[username].js]: Sure, Done 👍🏻

**Review by shubhanshu02 — APPROVED** [review]

**Pull request merged**